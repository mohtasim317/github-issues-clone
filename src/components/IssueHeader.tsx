import React, { FC } from "react";
import "../index.css";

const IssueHeader: FC = (props) => {
  return (
    <div className="issue-header">
      <h1>a</h1>
    </div>
  );
};

export default IssueHeader;
