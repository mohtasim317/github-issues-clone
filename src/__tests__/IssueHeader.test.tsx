import React from "react";
import renderer from "react-test-renderer";
import IssueHeader from "../components/IssueHeader";

describe("IssuesHeader Component Snapshot", () => {
  it("renders an IssueHeader component with lots of buttons and status updates on issues", () => {
    const component = renderer.create(<IssueHeader />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
