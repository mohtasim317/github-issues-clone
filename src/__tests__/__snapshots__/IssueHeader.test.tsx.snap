// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IssuesHeader Component Snapshot renders an IssueHeader component with lots of buttons and status updates on issues 1`] = `
<div
  className="issue-header"
>
  <div
    className="number-issues-container"
  >
    <a
      className="number-issues-open"
      href="/"
    >
      <svg
        aria-hidden="true"
        className="svg"
        version="1.1"
        viewBox="0 0 16 16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 1.5a6.5 6.5 0 100 13 6.5 6.5 0 000-13zM0 8a8 8 0 1116 0A8 8 0 010 8zm9 3a1 1 0 11-2 0 1 1 0 012 0zm-.25-6.25a.75.75 0 00-1.5 0v3.5a.75.75 0 001.5 0v-3.5z"
          fill-rule="evenodd"
        />
      </svg>
      <div
        className="issues-status-count"
      >
        460 Open
      </div>
    </a>
    <a
      className="number-issues-closed"
      href="/"
    >
      <svg
        aria-hidden="true"
        className="svg"
        version="1.1"
        viewBox="0 0 16 16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M13.78 4.22a.75.75 0 010 1.06l-7.25 7.25a.75.75 0 01-1.06 0L2.22 9.28a.75.75 0 011.06-1.06L6 10.94l6.72-6.72a.75.75 0 011.06 0z"
          fill-rule="evenodd"
        />
      </svg>
      <div
        className="issues-status-count"
      >
        8,973 Closed
      </div>
    </a>
  </div>
  <div
    className="issue-header-tabs-container"
  >
    <a
      className="issue-header-button issue-header-button-author "
      href="/"
    >
      Author↓
    </a>
    <a
      className="issue-header-button issue-header-button-label"
      href="/"
    >
      Label↓
    </a>
    <a
      className="issue-header-button issue-header-button-projects"
      href="/"
    >
      Projects↓
    </a>
    <a
      className="issue-header-button  issue-header-button-milestones"
      href="/"
    >
      Milestones↓
    </a>
    <a
      className="issue-header-button issue-header-button-assignee"
      href="/"
    >
      Assignee↓
    </a>
    <a
      className="issue-header-button issue-header-button-sort"
      href="/"
    >
      Sort↓
    </a>
  </div>
</div>
`;
